vulcanai2/models/dnn.py

```diff
@@ -122,11 +105,8 @@ def forward(self, x):
         :param x: input torch.Tensor
         :return: output torch.Tensor
         """
-        if self._input_network:
-            x = self._input_network(x)
 
-        if self._input_network and \
-            self._input_network.__class__.__name__ == "ConvNet":
+        if self._input_network and self._input_network.__class__.__name__ == "ConvNet":
```

Comment:
the backslash was done to shorten the length of that conditional for PEP8 reasons. why did you revert it?